src/core/web3/manager.ts

```diff
@@ -254,7 +254,15 @@ export class Web3Manager {
         );
       }
       this.#client = this.#config.wagmiOptions.wagmiClient as WagmiClient;
-      this.#connectors = this.#client.connectors;
+
+      // Show our four default connectors if only an injected connector is passed in
+      // (this is wagmi's default when no connectors are passed into the wagmi client)
+      if (
+        this.#client.connectors.length !== 1 ||
```

Comment:
If the length is 0 the next line will fail so we need to be a bit defensive here

Reply:
wagmi doesn't allow it to be zero but I'll update in case they change

Reply:
actually, if it is `0`, it'll be true and skip the `connectors[0].id` line